client/app/models/collection.js

```diff
@@ -7,6 +7,7 @@ export default DS.Model.extend({
     created_by: DS.belongsTo('user'),
     dateCreated: DS.attr('date'),
     dateUpdated: DS.attr('date'),
+    settings: DS.attr('object'),
```

Comment:
Ok, I noticed this because I'm getting a 400 error when I try to edit a collection, and it's because settings is an object in the request body. Can you do what you did when creating a collection and convert that value to a string?